app/v1/controllers/CreateAndAmendReliefInvestmentsController.scala

```diff
@@ -106,10 +106,19 @@ class CreateAndAmendReliefInvestmentsController @Inject() (val authService: Enro
   private def errorResult(errorWrapper: ErrorWrapper) = {
 
     errorWrapper.error match {
-      case NinoFormatError | BadRequestError | TaxYearFormatError | RuleIncorrectOrEmptyBodyError | RuleTaxYearNotSupportedError |
-          RuleTaxYearRangeInvalidError | MtdErrorWithCustomMessage(ValueFormatError.code) | MtdErrorWithCustomMessage(
-            DateOfInvestmentFormatError.code) | MtdErrorWithCustomMessage(NameFormatError.code) | MtdErrorWithCustomMessage(
-            UniqueInvestmentRefFormatError.code) =>
+      case _
+          if errorWrapper.containsAnyOf(
+            NinoFormatError,
+            BadRequestError,
```

Comment:
Curious as to why we tend to include "BadRequestError" in the errorWrapper mappings. It's not included in the specs so surely all required error scenarios are covered by the other errors?

Reply:
The [requestParser](https://github.com/hmrc/individuals-reliefs-api/blob/main/app/v1/controllers/requestParsers/RequestParser.scala#L40) class can return a BadRequestError in the case where there a multiple validation errors.

Reply:
It's in the spec under the `Multiple validation type error` section.